src/app/pages/portal/panels/feature/feature-info/feature-info.module.ts

```diff
@@ -8,6 +8,7 @@ import { IgoLanguageModule } from '@igo2/core';
 import { IgoSearchResultsModule } from '@igo2/geo';
 import { IgoFeatureModule } from '../feature.module';
 import { FeatureInfoComponent } from './feature-info.component';
+import { IgoFeatureModule as IgoFeatureModules } from '@igo2/geo';
```

Comment:
On pourrait renommer le `IgoFeatureModule` de IGO2-QUEBEC  `AppFeatureModule` et conserver la nomenclature de la librairie.
```suggestion
import { AppFeatureModule } from '../feature.module';
import { IgoFeatureModule } from '@igo2/geo';
```